src/gui/optionsdlg.cpp

```diff
@@ -648,6 +649,7 @@ void OptionsDialog::saveOptions()
     pref->setWebUiEnabled(isWebUiEnabled());
     if (isWebUiEnabled()) {
         pref->setServerDomains(m_ui->textServerDomains->text());
+        pref->setWebUiAddress(webUiAddress());
```

Comment:
You can just inline it.

ps. Everything else is fine.

Reply:
The line below calls `webUiPort()` so I copied that idiom. If one is changed so should the other no?

Let me know what is preferred here.

Reply:
>The line below calls webUiPort() so I copied that idiom. If one is changed so should the other no?

Yes, if you're willing to refactor it.

>Let me know what is preferred here.

If other devs don't object inlining it, let's wait a bit.

Reply:
There is no reason for having additional getters for it.

Reply:
Ok I pushed this as a second commit.